aiobotocore/config.py

```diff
@@ -3,13 +3,18 @@
 import botocore.client
 from botocore.exceptions import ParamValidationError
 
+from aiobotocore.httpsession import AIOHTTPSession
+
 
 class AioConfig(botocore.client.Config):
-    def __init__(self, connector_args=None, **kwargs):
+    def __init__(
+        self, connector_args=None, http_session_cls=AIOHTTPSession, **kwargs
```

Comment:
Since create_endpoint has a non-None default we'd need to do:
```py
def __init__(self, ..., http_session_cls=None):
  ...
  if http_session_cls is None:
    http_session_cls_kwargs = {}
  else:
    http_session_cls_kwargs = {"http_session_cls": http_session_cls}
  ...
  create_endpoint(
    ...
    **http_session_cls_kwargs
  )
```
to not override the default by passing `None`.

Second option would be to also default the `http_session_cls` parameter to `None` in `AioEndpointCreater.create_endpoint` and then
```py
http_session_cls = AIOHTTPSession if http_session_cls is None
```
Minor downside would be that we don't communicate the default to end-users in the signature.

Third option would be a global variable `HTTP_SESSION_CLS_DEFAULT` that both methods use.

Reply:
oh wait, you said `args.py`... That's a third place for it, which makes me want to do a global constant defined ... somewhere. `aiobotocore/endpoint.py`?

Reply:
oh man, heh, yea global is fine too